nss-day-cohort-60/rare-client-tenacious-tators
# Jes user styling

# Description of Proposed Changes

Added styling and follow/unfollow buttons to UserManagement
Added styling to User Details
---


# Steps to Test

1. Fetch the jes-user-styling branch and switch to it 
2. Fetch the SERVER branch jes-author-view and switch to it, run debugger
3. Navigate to /users verify that everything looks pretty, follow and unfollow buttons work (should toggle when clicked)
4.navigate to user details page, verify that it looks pretty, that follow/unfollow works, that date is formatted.
